components/Tabs/Tabs.js

```diff
@@ -1,51 +1,72 @@
+class Tabs {
+  constructor(element) {
+    this.element = element
+
+    //grabbing all tab links
+    this.tabLinks =  document.querySelectorAll('.tabs-link').forEach(function(section) {
+      return new TabLink(section);
+    });
+  }
+
+  deselect() {
+    
+  }
+}
+
 
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = document.querySelector(`.tabs-link[data-tab='${this.element.dataset.tab}']`);
```

Comment:
just a reminder of what we did in standup
```suggestion
    this.data = this.element.dataset.tab
```